app/models/installed_technology.rb

```diff
@@ -214,10 +214,18 @@ def as_json(*)
   end
 
   def yearly_investment
-    om_costs_per_year.to_f                   +
-    initial_investment.to_f                  / (technical_lifetime || 1) +
-    om_costs_per_full_load_hour.to_f         / (full_load_hours || 1) +
-    om_costs_for_ccs_per_full_load_hour.to_f / (full_load_hours || 1)
+    (initial_investment.to_f / (technical_lifetime || 1)) + yearly_om_cost
+  end
+
+  def yearly_om_cost
+    om_costs_per_year.to_f + (
+      om_costs_per_full_load_hour.to_f +
+      om_costs_for_ccs_per_full_load_hour.to_f
+    ) / yearly_full_load_hours
+  end
```

Comment:
I think the solution you ended up with – moving variable costs to a separate method – was the nicest; it avoids having to think about which value is being divided by `yearly_full_load_hours`.

Maybe something like this would also have been okay (all it does is move the opening bracket down a line to help emphasise that the fixed cost is not divided by the full load hours):

```ruby
def yearly_om_cost
  om_costs_per_year.to_f +
    ( om_costs_per_full_load_hour.to_f +
      om_costs_for_ccs_per_full_load_hour.to_f
    ) / yearly_full_load_hours
end
```

The original version passes [Rubocop](https://github.com/bbatsov/rubocop), so I think it's just personal preference. :smiley: